src/mpi/romio/adio/ad_lustre/ad_lustre_wrcoll.c

```diff
@@ -1106,7 +1106,7 @@ static void ADIOI_LUSTRE_IterateOneSided(ADIO_File fd, const void *buf, int *str
      * aggregator.  The segmentLen is the maximum size in bytes of each segment
      * (stripeSize*number of aggs).  Iteratively call ADIOI_OneSidedWriteAggregation
      * for each segment to aggregate the data to the collective buffers, but only do
-     * the actual write (via flushCB stripe parm) once stripesPerAgg stripes
+     * the actual write (via flushCB stripe param) once stripesPerAgg stripes
```

Comment:
This is an odd one. The struct is named `stripeParms`, which appears to be short for stripe parameters. So I think either param is technically correct, but parm is better in context.

Reply:
I have whitelisted `parm`, I don't know how this one leaked through. Anyway, I thought `parm` is just an abbreviation of `param` which is an abbreviation of `parameter`. Is `parm` more preferred or does it have a different meaning?

Meanwhile, let me clean up that instance.

Reply:
Personally I prefer "param", but the code in this instance historically uses "parm". I suggest we match the code instead of changing variable names in this instance.

Reply:
Oh, I *manually* did the quoted fix before I decided to whitelist `parm`. So the script is working after all. I have reverted the wrong *manual* change.